examples/rvc-app/rvc-common/rvc-app.matter

```diff
@@ -1552,7 +1552,7 @@ provisional cluster ServiceArea = 336 {
     int32u areaID = 0;
     OperationalStatusEnum status = 1;
     optional nullable elapsed_s totalOperationalTime = 2;
-    optional nullable elapsed_s estimatedTime = 3;
+    optional nullable elapsed_s initialTimeEstimate = 3;
```

Comment:
Sorry, I didn't realize there was on ongoing discussion about this, but it looks like the RVC folks are going to update the spec to fix this rather than the SDK. 

See https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/11038. I'm going to try to get them to land that soon, after that point, this will need to be regenerated.

Reply:
Thanks for the heads up, yeah that change fixed some issues with this PR. Just updated the PR with the latest changes from the specs.